docs/wegue-configuration.md

```diff
@@ -226,6 +227,29 @@ Below is an example for such a configuration object:
     }
 ```
 
+### mapHover
+
+Optional configuration object to customize the behavior of hover tooltips on the map. 
+The following properties are supported:
+
+| Property           | Meaning   | Example |
+|--------------------|:---------:|---------|
+| delay              | Timespan in milliseconds, by which displaying the tooltip is deferred after the mouse pointer rests. Defaults to `150`. | `"delay": 150`
+| hideOnMousemove    | Hide the tooltip when the mouse cursor is moved. Defaults to `false`. | `"hideOnMousemove": false`
+| hoverOverlay       | ID of a custom map overlay to use as a default display when a feature of the layer is hovered. Declaration of the `hoverOverlay` property on a mapLayer level takes precedence. For more information on how to implement a map overlay see the [reusable components](reusable-components?id=map-overlay) section. Defaults to Wegue's default tooltip `wgu-hover-tooltip` |  `"hoverOverlay": "my-custom-overlay"`
```

Comment:
Again a detail here, I personally think "when a feature of **the** layer is hovered" is a bit inaccurate.

It would be more correct with something like "...when a feature of **a hoverable layer** is hovered" which is not really great as we have two times the **hover** word very close together.  
I first thought about "...when a feature of **a hoverable layer** is under the mouse pointer" which once again becomes vague and perhaps too verbose... if you have a better idea...

Reply:
Good job, this will be clearer for end users as you wrote it !